maptivist/src/main/webapp/script.js

```diff
@@ -297,4 +308,14 @@ var GoogleAuth;
 
   function updateSigninStatus(isSignedIn) {
     setSigninStatus();
-  }
\ No newline at end of file
+  }
+
+async function getVote() {
+    const response = await fetch("/votes", {method: "GET"});
```

Comment:
If you did getElementById("vote-button") you could get the ID and pass it to the GET request as a parameter. That way you could look up the entity by id from GET.